06-statistical-testing.Rmd

```diff
@@ -120,25 +122,25 @@ svyttest(formula,
-  b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.
+1. **One-sample t-test:**     
+  a. **Comparison to 0:** `var ~ 0`, where `var` is the measure of interest, and `0` is the value we compare it to. For example, we could test if the proportion of the population that has blue eyes is different from `0`.  
+  b. **Comparison to a different value:** `I(var - value) ~ 0`, where `var` is the measure of interest and `value` is what we are comparing to. We need to use the `I()` function to tell the program to calculate the difference between the variable and the comparison value before testing. For example, we could test if the proportion of the population that has blue eyes is different from `25%` by using `I(var - 0.25)~0`.  
+2. **Two-sample t-test:**  
+  a. **Unpaired:**  
+    - **2 level grouping variable:** `var ~ groupVar`, where `var` is the measure of interest and `groupVar` is a variable with two categories. For example, we could test if the proportion of the population that has blue eyes is different for children aged 5-10 years old compared to children under 5 years old.   
+    - **3+ level grouping variable:** `var ~ I(groupVar == level)`, where `var` is the measure of interest, `groupVar` is the categorical variable, and `level` is the category level to isolate. Again we need to use the `I()` function to tell the program to isolate the category before doing the comparison across groups. For example, we could test if the test scores in one classroom differed from all other classrooms.   
+  b. **Paired:** `I(var_1 - var_2) ~ 0`, where `var_1` is the first variable of interest and `var_2` is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against `0`. For example, we could test if test scores on a subject differed between the start and the end of a course.  
 
 Additionally, the `na.rm` argument defaults to `FALSE`, which means if any data is missing, the t-test will not compute. Throughout this chapter we will always set `na.rm = TRUE`, but before analyzing the survey data, review the notes provided in Chapter \@ref(c04-understanding-survey-data-documentation) to better understand how to handle missing data. Finally, the `level` argument is $1-\alpha$, or the amount of type 1 error. The default is $0.95$.
```

Comment:
Is this `level` different than the one above? "3+ level grouping variable: var ~ I(groupVar == level), where var is the measure of interest, groupVar is the categorical variable, and level is the category level to isolate"